src/TwigComponent/tests/Unit/ComponentAttributesTest.php

```diff
@@ -87,4 +87,22 @@ public function testCanAddStimulusController(): void
             'data-foo-name-value' => 'ryan',
         ], $attributes->all());
     }
+
+    public function testBooleanBehaviour(): void
+    {
+        $attributes = new ComponentAttributes(['disabled' => true]);
+
+        $this->assertSame(['disabled' => true], $attributes->all());
+        $this->assertSame(' disabled', (string) $attributes);
+
+        $attributes = new ComponentAttributes(['disabled' => false]);
+
+        $this->assertSame(['disabled' => false], $attributes->all());
+        $this->assertSame('', (string) $attributes);
```

Comment:
Both `true` and `false` behaviors are BC breaks, no? Bug fixes, but breaks?

Reply:
Yes, not sure how we should handle this...

Reply:
Actually, these are ok.

Currently:

```
true => disabled="1"
false => disabled
```

So changing `true` to just `disabled` looks NOT like a BC break to me/
The previous `false` was just plain incorrect. That’s a bug fix